### ISSUE TITLE
Create an interactive document

### ISSUE THREAD

**Issue by ccamara** [WarwickCIM/quarto-workshop#15]
This task consists of creating a document containing some kind of interactivity based on user's input. It can be a visualisation or anything you fancy, as long as it changes based on some inputs.
You can use any solution (See documentation below) or, if you feel brave and have enough time, try more than one (will be great for discussion later!)

## Steps to reproduce:
 
1. Create a `qmd`file containing some code blocks that need external libraries
2. Create an html output  that changes based on user's inputs. Y

## Expected outputs

1. HTML document (it can be an Article, slides, ...) that changes based on user's inputs
2. Documentation (please use [`documentation_template.qmd`](https://github.com/WarwickCIM/quarto-workshop/blob/main/tasks/template/documentation_template.qmd) as a template)

## Guides

* https://quarto.org/docs/interactive/
* https://neocarto.github.io/Quarto/

**Comment by jack89roberts** [comment]
I'd like to give this one a go 🙂 (P.S. I don't believe I have permissions to assign myself)

**Comment by nicola-martin** [comment]
I'd like to try this one

**Comment by schndo** [comment]
I'm interested in this task, too :)

**Comment by alessandrofelder** [comment]
I'd be interested in this :)

**Comment by ccamara** [comment]
> I'd like to give this one a go slightly_smiling_face (P.S. I don't believe I have permissions to assign myself)

Yes, you're right. No problem though. Showing interest via comment works well, too. Thanks for your interest
